integreat_cms/cms/migrations/0021_rename_location_not_on_map_poi_location_on_map.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 3.2.12 on 2022-04-03 14:22
```

Comment:
I think this filename is a bit cumbersome, maybe `0021_rename_location_not_on_map.py` would be sufficient?

Reply:
sure, hang on :) i had some issues with the translation file and pre-commit hooks anyway, but I think I fixed in now